xtask/src/main.rs

```diff
@@ -191,17 +195,17 @@ fn kasm(profile: Build) -> Result<()> {
 
 fn dist(profile: Build) -> Result<()> {
     build(profile)?;
-    let status = Command::new(objcopy())
-        .arg("--input-target=elf64-x86-64")
-        .arg("--output-target=elf32-i386")
-        .arg(format!("target/{}/{}/r9", target(), profile.dir()))
-        .arg(format!(
-            "target/{}/{}/r9.elf32",
-            target(),
-            profile.dir()
-        ))
-        .current_dir(workspace())
-        .status()?;
+    let mut cmd = Command::new(objcopy());
+    cmd.arg("--input-target=elf64-x86-64");
+    cmd.arg("--output-target=elf32-i386");
+    cmd.arg(format!("target/{}/{}/r9-x86_64", target(), profile.dir()));
+    cmd.arg(format!(
+        "target/{}/{}/r9-x86_64.elf32",
```

Comment:
Just call the _output_ `r9.elf32`. The architecture (and ABI, build profile, etc) are all in the target subdirectory.